package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "reservations",
+  "version": "1.0.0",
+  "description": "reservation module for mock up Open Table page",
+  "main": "./server/index.js",
+  "scripts": {
+    "test": "node_modules/jest/bin/jest.js --watch",
```

Comment:
@LittleJennie 

You may want to look into NPX and running commands via files from your node modules here, if you're interested in another way to do this!